src/netex_idf/stops.rs

```diff
@@ -188,10 +249,19 @@ fn load_stop_points<'a>(
             stop_point
         };
 
+        if let Some(associated_equipment) =
+            get_or_create_equipment(quay, &mut equipments, &mut id_incr)
+        {
+            stop_point.equipment_id = Some(associated_equipment.id.clone());
+        }
+
         stop_points.push(stop_point)?;
     }
 
-    Ok(stop_points)
+    let mut equipments: Vec<_> = equipments.into_iter().map(|(_, e)| e).collect();
+    equipments.sort_unstable_by(|tp1, tp2| tp1.id.cmp(&tp2.id));
```

Comment:
It's really annoying we have to do that... even if I don't know how to do it differently in this case...